nailgun/entities.py

```diff
@@ -2740,17 +2870,41 @@ def read(self, entity=None, attrs=None, ignore=()):
         return super(PuppetModule, self).read(entity, attrs, ignore)
 
 
-class Realm(Entity):
+class Realm(
```

Comment:
I did manage to create and work with a 'Realm' entity via the API. Let me spend a few minutes working on this, and I'll see if I can post some output showing how I did it.